dotcom-rendering/src/web/components/MostViewedFooterData.importable.tsx

```diff
@@ -2,17 +2,29 @@ import { css } from '@emotion/react';
 
 import { border, from, Breakpoint } from '@guardian/source-foundations';
 
+import { useAB } from '@guardian/ab-react';
 import { useApi } from '../lib/useApi';
 import { joinUrl } from '../../lib/joinUrl';
 import { decideTrail } from '../lib/decideTrail';
 
 import { MostViewedFooterGrid } from './MostViewedFooterGrid';
 import { MostViewedFooterSecondTierItem } from './MostViewedFooterSecondTierItem';
+import { abTestTest } from '../experiments/tests/ab-test-test';
+import { WithABProvider } from './WithABProvider';
+
+type WithABProps = {
+	sectionName?: string;
+	palette: Palette;
+	ajaxUrl: string;
+};
 
 type Props = {
 	sectionName?: string;
 	palette: Palette;
 	ajaxUrl: string;
```

Comment:
You could use `extends` here to reduce type duplication 🙂

Reply:
Great idea